go/libraries/doltcore/remotestorage/chunk_store.go

```diff
@@ -440,7 +440,7 @@ func (dcs *DoltChunkStore) HasMany(ctx context.Context, hashes hash.HashSet) (ha
 				absent[currHash] = struct{}{}
 				j++
 			} else {
-				c := chunks.NewChunkWithHash(currHash, []byte{})
+				c := nbs.ChunkToCompressedChunk(chunks.NewChunkWithHash(currHash, []byte{}))
```

Comment:
should use chunks.EmptyCompressedChunk

Reply:
But I need the hash, right? Left it like this...